tests/blockchain/eth/contracts/main/miners_escrow/test_mining.py

```diff
@@ -294,3 +299,391 @@ def test_mining(testerchain, token, escrow_contract):
     assert 6 == len(lock_log.get_all_entries())
     assert 1 == len(divides_log.get_all_entries())
     assert 10 == len(activity_log.get_all_entries())
+
+
+@pytest.mark.slow
+def test_slashing(testerchain, token, escrow_contract):
```

Comment:
This is great. @jMyles Time for an integration test with an evil Ursula, don't you think?